kyo-prelude/shared/src/main/scala/kyo/Env.scala

```diff
@@ -102,4 +99,10 @@ object Env:
     /** Creates a UseOps instance for a given type. */
     inline def use[R >: Nothing]: UseOps[R] = UseOps(())
 
+    private val cachedIsolate                         = Isolate.Contextual.derive[Env[Any], Any]
+    given isolate[V]: Isolate.Contextual[Env[V], Any] = cachedIsolate.asInstanceOf[Isolate.Contextual[Env[V], Any]]
+
+    given eliminateEnv: Reducible.Eliminable[Env[Any]] with {}
+    private inline def erasedTag[R] = Tag[Env[Any]].asInstanceOf[Tag[Env[R]]]
```

Comment:
I'll follow up moving this kind of implicit evidences to the end of files. I think keeping at the top isn't great because it's the first thing people will read in the source file and takes away the focus on the provided features.